### ISSUE TITLE
Classic OOP style?

### ISSUE THREAD

**Issue by rubyFeedback** [jruby/jrubyfx#131]
If I understand jrubyfx correctly then it is a DSL, where we use verbs to describe
what we want to have e. g.:

    vbox {
    }

And so forth.

Via ruby-gtk I am more used to:

    x = Name.new

And then use .add() or << to chain widgets together.

Would it be possible for jrubyfx to also support that style? Where e. g. classes
from JavaFX are mapped 1:1? Like:  JavaFX::TextArea.new or whatever the
name is?

I understand that in ruby everyone loves DSLs but I found them a lot more
complicated than classical OOP.

**Comment by byteit101** [comment]
Yes, that style should be supported as it's the style I prefer. See some of
the non-dsl samples in fxml or contrib.
